hw/ip/csrng/data/csrng.prj.hjson

```diff
@@ -5,10 +5,10 @@
 {
     name:               "csrng",
     design_spec:        "hw/ip/csrng/doc",
-    dv_doc:            "hw/ip/csrng/doc/dv",
+    dv_doc:             "hw/ip/csrng/doc/dv",
     hw_checklist:       "hw/ip/csrng/doc/checklist",
     version:            "0.5",
     life_stage:         "L1",
     design_stage:       "D1",
-    verification_stage: "V0",
+    verification_stage: "V1",
```

Comment:
perhaps move this update to a separate PR. And this one usually goes with some update in checklist.md